Add intro email template to send to interns (#145)
jupyterhub/outreachy: markdown-templates/intern-intro-email-template.md

# Email template: Intern Introduction

> Dear {APPLICANT NAME},

Author: ```suggestion
Dear {INTERN NAME},
```

Welcome to the JupyterHub community! We are so excited to have you here and begin
working with you on your Outreachy project: **{PROJECT TITLE}**. This email will
put you in touch with your primary mentor {MENTOR NAME}, if you are not already,
and provide some instructions for getting started in your first week.

## Meeting Notes

We have provided a document for you to keep notes from your mentor meetings in.
It's very important that you keep notes so that you can remember what was
discussed, which may help in planning next issues for you to work on or writing
up blog posts.

- Notes link: {NOTES LINK}

This layout is for **guidance only**. If you prefer a different format, you are
free to change the document to suit you.

In the JupyterHub community, we use HackMD (https://hackmd.io) for collaborative
notetaking, which is the platform your notes document has been created on. If
you are new to HackMD, you can read this short guide from Open Life Science on

> how to use it: https://hackmd.io/@openlifescience/OLS-HackMD-guide However, this
> also is for guidance only, and if using another platform, like Google Docs,

Author: ```suggestion
how to use it: https://hackmd.io/@openlifescience/OLS-HackMD-guide However, this
is also for guidance only, and if using another platform, like Google Docs,
```

works better - that's ok too!

## Setup your Mentor Meetings

Please reach out to your mentor (cc'd on this email) and arrange a suitable
recurring day, time, and platform for your mentor meetings.

- **Day and Time:** Remember that you and your mentor may not be in the same
  timezone, so try to be considerate of early mornings or late evenings. We also
  highly encourage taking the weekends off to rest and recharge!
- **Frequency:** Outreachy recommends meeting with your mentor weekly for about
  30 minutes, so make sure the day and time you pick is one that is generally

>   available every week.

Author: I need to verify this claim, but the outreachy site is currently down with certificate issues

- **Platform:** How will you and your mentor connect? Zoom, Google Meet, Jitsi,
  something else? Maybe something not video-based? Make sure you decide this
  with your mentor and generate a link that you can reuse for each meeting.

Once you have established all these details, **add them to the top of your notes
document so you can always find them!**

<!-- This section is optional depending on if someone is able to lead the demo -->

## JupyterHub Demo Session

In the first week, we will like to host a Demo Session of the JupyterHub platform
to explain how it works and you have the opportunity to ask any questions before
starting on your projects. Please fill in the {SCHEDULING TOOL} link below with your
availability to attend this session next week. We will distribute an event invite
by the end of the week.

- {SCHEDULING TOOL} link: {LINK}

## Tell us about yourself for a welcome blog

We would like to post to the Jupyter blog (https://blog.jupyter.org) to tell the

> community a little bit about yourselves and what you'll be working on Could you
> please provide a few short sentences covering:

Author: ```suggestion
community a little bit about yourselves and what you'll be working on. Could you
please provide a few short sentences covering:
```

Please send this **directly to {NAME AND EMAIL}**, rather than replying to this
thread.

## Code of Conduct and Reporting Issues

We hope you have and will strive to provide a positive experience while you are
working with JupyterHub. But just in case...

Anyone working in JupyterHub is bound by Project Jupyter's Code of Conduct, which
is available to read here: https://jupyter.org/governance/conduct/code_of_conduct.html
This should give you an idea of what is acceptable and unacceptable behaviour
within our community.

If you experience or witness anything that you believe breaks the Code of Conduct,
or if you have any general worries about your Outreachy project, you can take the
following actions:

1. In the first instance, please reach out to your primary mentor: {NAME, EMAIL}
2. If you are not able to discuss your issue with your primary mentor, please
   reach out to the JupyterHub Community Coordinator: {NAME, EMAIL}
3. If you believe the Community Coordinator has a conflict of interest regarding
   your issue, you can reach out to our co-mentor(s): {NAME, EMAIL}
4. Finally, if you do not feel that the issue can be resolved within the JupyterHub

>    community, you can contact the Outreachy organisers: {LINK}

Author: ```suggestion
   community, you can contact the Outreachy organisers: organizers@outreachy.org
```

>    community, you can contact the Outreachy organisers: {LINK}

Author: Link TBA when the outreachy website is back up
